.github/workflows/manual_build_and_push_docker_image.yml

```diff
@@ -0,0 +1,31 @@
+name: Manually build a Docker image for a given tag and push to Docker hub
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: Tag to build and push an image for
+        required: true
+
+jobs:
+  build_and_push:
+    name: Build a new Docker image for the given tag and push it to Docker
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
How do you control what tag gets checked out? Looking at similar [task](https://github.com/Signalen/build-and-push/blob/master/.github/workflows/dockerhub-manual.yml) at VNG  and `actions/checkout@v2` [documentation](https://github.com/actions/checkout/tree/7884fcad6b5d53d10323aee724dc68d8b9096a2e) I think you should use `ref: ${{ github.event.inputs.tag }}` to checkout the actual tag you are interested in and not just the SHA of whatever the repository was at when the workflow was triggered.
```
    steps:
      - uses: actions/checkout@v2
        with: 
          ref: ${{ github.event.inputs.tag }}
```

Reply:
I looked at the VNG task ;) and I just forgot to add this. Do you know of a way to tests github actions when they are still in a branch?